src/main/java/redis/clients/jedis/commands/StringCommands.java

```diff
@@ -12,70 +12,442 @@
+   * @param key
+   * @param start
+   * @param end
+   * @return Bulk reply
+   */
   String substr(String key, int start, int end);
 
+  /**
+   * <b><a href="http://redis.io/commands/strlen">StrLen Command</a></b>
+   * Return the length of the string value stored at key.
+   * <p>
+   * @param key
+   * @return Integer reply, specifically the length of the string at key, or 0 when key does not exist
+   */
   long strlen(String key);
 
+  /**
+   * <b><a href="http://redis.io/commands/bitcount">Bitcount Command</a></b>
+   * Count the number of set bits (population counting) in a string.
+   * <p>
+   * @param key
+   * @return Integer reply, specifically the number of bits set to 1
+   */
   long bitcount(String key);
 
+  /**
+   * <b><a href="http://redis.io/commands/bitcount">Bitcount Command</a></b>
+   * Count the number of set bits (population counting) in a string only in an interval start and end.
+   * <p>
+   * Like for the GETRANGE command start and end can contain negative values in order to index bytes
+   * starting from the end of the string, where -1 is the last byte, -2 is the penultimate, and so forth.
+   * <p>
+   * @param key
+   * @param start byte start index
+   * @param end byte end index
+   * @return Integer reply, specifically the number of bits set to 1
+   */
   long bitcount(String key, long start, long end);
 
+  /**
+   * @see StringCommands#bitcount(String, long, long) with BYTE|BIT option.
+   * @param key
+   * @param start byte start index
+   * @param end byte end index
+   * @param option {@link BitCountOption}
+   * @return Integer reply, specifically the number of bits set to 1
+   */
   long bitcount(String key, long start, long end, BitCountOption option);
 
+  /**
+   * <b><a href="http://redis.io/commands/bitpos">Bitpos Command</a></b>
+   * Return the position of the first bit set to 1 or 0 in a string.
+   * <p>
+   * @param key
+   * @param value the bit value
+   * @return The position of the first bit set to 1 or 0 according to the request
+   */
   long bitpos(String key, boolean value);
 
+  /**
+   * <b><a href="http://redis.io/commands/bitpos">Bitpos Command</a></b>
+   * Return the position of the first bit set to 1 or 0 in a string.
+   * <p>
+   * @param key
+   * @param value the bit value
+   * @param params {@link BitPosParams}
+   * @return The position of the first bit set to 1 or 0 according to the request
+   */
   long bitpos(String key, boolean value, BitPosParams params);
 
+  /**
+   * <b><a href="http://redis.io/commands/bitfield">Bitfield Command</a></b>
+   * The command treats a Redis string as an array of bits, and is capable of addressing specific integer
+   * fields of varying bit widths and arbitrary non (necessary) aligned offset.
+   * <p>
+   * @param key
+   * @param arguments
+   * @return A List of results
+   */
   List<Long> bitfield(String key, String...arguments);
 
+  /**
+   * The readonly version of {@link #bitfield(String, String...) BITFIELD}
```

Comment:
full name

Reply:
Hope that this is the last one, because I can't find more